pkg/kubectl/cmd/apply_view_last_applied.go

```diff
@@ -73,6 +74,7 @@ func NewCmdApplyViewLastApplied(f cmdutil.Factory, out, err io.Writer) *cobra.Co
 
 	cmd.Flags().StringP("output", "o", "", "Output format. Must be one of yaml|json")
 	cmd.Flags().StringVarP(&options.Selector, "selector", "l", "", "Selector (label query) to filter on, supports '=', '==', and '!='.")
+	cmd.Flags().BoolVar(&options.all, "all", false, "[-all] to select all the specified resources.")
```

Comment:
Don't add "[-all]" to the flag description, make it just something like "select all resources of the specified resource types".